src/seedu/addressbook/commands/EditCommand.java

```
@@ -0,0 +1,11 @@
+package seedu.addressbook.commands;
+
+public class EditCommand extends Command {
+
+	@Override
+	public CommandResult execute() {
+		// TODO Auto-generated method stub
+		return null;
```

Consider throwing a RuntimeException or something instead of just returning a bogus value. At this point in time, the very act of calling this method should already be considered an error as this method is still unfinished.

---

Although this would not be a problem if you ordered your commits such that you do not have to add all these `// TODO` all over the place